src/lib/LUA/rx_devLUA.cpp

```diff
@@ -25,7 +25,7 @@ static const char *rxModes = "50Hz;60Hz;100Hz;160Hz;333Hz;400Hz;10kHzDuty;On/Off
 static struct luaItem_selection luaSerialProtocol = {
     {"Protocol", CRSF_TEXT_SELECTION},
     0, // value
-    "CRSF;Inverted CRSF;SBUS;Inverted SBUS;SUMD;DJI RS Pro;HoTT Telemetry;Spektrum Satellite",
+    "CRSF;Inverted CRSF;SBUS;Inverted SBUS;SUMD;DJI RS Pro;HoTT Telemetry;Spektrum Remote Receiver",
```

Comment:
Are you aware that on BW screens only about 12 characters will be seen so maybe "Spektrum RR", also as mentioned previously (AFAIRC) why not just move all DSM~ entries here, like it is with CRSF, SBUS?